src/virtualenv/seed/embed/base_embed.py

```diff
@@ -18,14 +20,21 @@ def __init__(self, options) -> None:
 
         self.pip_version = options.pip
         self.setuptools_version = options.setuptools
-        self.wheel_version = options.wheel
 
         self.no_pip = options.no_pip
         self.no_setuptools = options.no_setuptools
-        self.no_wheel = options.no_wheel
         self.app_data = options.app_data
         self.periodic_update = not options.no_periodic_update
 
+        if options.no_wheel:
+            warn(
+                "The --no-wheel option is deprecated. "
+                "It has no effect, wheel is no longer bundled in virtualenv. "
+                "This option will be removed in pip 26.",
```

Comment:
in pip 26? should this be a virtualenv version?

Reply:
Yeah, it should 🤔 @stefanor

Reply:
But I think here we're conflating virtualenv wheel support and pip wheel support 🤔

Reply:
#2876